sandpack-react/src/components/index.ts

```diff
@@ -7,3 +7,4 @@ export * from "./Preview";
 export * from "./TranspiledCode";
 export * from "./Tests";
 export * from "./Console";
+export * from "./ReactDevTools";
```

Comment:
`ReactDevTools` is not currently working, that's why it hasn't been exported

Reply:
yes, i miss that)